content/en/monitors/status/events.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Status Events
+further_reading:
+- link: "service_management/events/"
+  tag: "Documentation"
+  text: "Event Management"
+---
+
+<div class="alert alert-info">Status Events is part of the <a href="/monitors/status/status_page">provisional Monitor Status Page</a>. If you are using the legacy status page, see the <a href="/monitors/status/status_legacy">Status Page (Legacy)</a> documentation.</div>
+
+## Overview
+
+{{< img src="/monitors/status/status_page_event_details.png" alt="Monitor status page displaying event details" style="width:100%;" >}}
+
+All events generated by your monitor appear on the status page, showing the groups' name, event type, and timestamp. The Event timeline also includes downtime and audit trail events.
```

Comment:
```suggestion
All events generated by your monitor appear on the monitor's status page, showing the groups' name, event type, and timestamp. The Event timeline also includes downtime and audit trail events.
```
Or "monitor status page," but in this case I was trying to make it clear that each monitor has its own status page.